utils/oscap-tool.h

```diff
@@ -118,6 +118,7 @@ struct oscap_action {
 	char *f_verbose_log;
 	/* others */
         char *profile;
+        const char *rule;
```

Comment:
You mean that indents should be tabs? Because this whole structure uses a mix of tabs and spaces for indentation, I was not sure which one to use and AFAIK there is no coding guidelines for openscap project, right?